### DISCUSSION TITLE
How can i add subtitles to video player

### DISCUSSION THREAD

**Discussion by issam-seghir** [muxinc/next-video#264] [Q&A] [answered]
I have AI-generated subtitles ready in the backend and I want to add them to the video. I've read the documentation and there's nothing about subtitles and captions!

```js
import Video from 'next-video';
import awesomeVideo from 'https://www.mydomain.com/remote-video.mp4';
 
export default function Page() {
  return <Video src={awesomeVideo} />;
}
```

**Comment by luwes** [comment, marked as answer]
with Mux you can add them via the Mux video API
https://docs.mux.com/api-reference#video/operation/create-asset-track

or if you like to add them manually from your own backend you can add a `<track>` element.
it works similarly to how it works with the native `<video>` tag.
